src/hotspot/share/classfile/systemDictionaryShared.cpp

```diff
@@ -1415,6 +1415,10 @@ InstanceKlass* SystemDictionaryShared::find_builtin_class(Symbol* name) {
   if (record != NULL) {
     assert(!record->_klass->is_hidden(), "hidden class cannot be looked up by name");
     assert(check_alignment(record->_klass), "Address not aligned");
+    // The regenerated lamdbda Holder classes are not saved, should avoid call CFLH
```

Comment:
I think it will be clear to say
```
// We did not save the classfile data of the regenerated LambdaForm invoker classes,
// so we cannot support CLFH for such classes. 
```